### PR TITLE
Adding rados test case CEPH-9228

### PR DESCRIPTION
**Pull request by tintumathew10**
Signed-off-by: tintumathew10 <tmathew@redhat.com>

Adding rados test case CEPH-9228.

logs: http://magna002.ceph.redhat.com/ceph-qe-logs/tmathew/cephci-run-77IXNX/
http://magna002.ceph.redhat.com/ceph-qe-logs/tmathew/cephci-run-V8ADTP/

### CODE REVIEW

File: suites/pacific/rados/tier-3_rados_test-osd-inprogress-rebalance.yaml [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+              rados_put: true
+              byte_size: 1024
+              osd_max_backfills: 16
+              osd_recovery_max_active: 16
+          - create_pool:
+              create: true
+              pool_name: ercpool_1
+              pool_type: erasure
+              pg_num: 32
+              k: 4
```
Comment (pdhiran) [inline suites/pacific/rados/tier-3_rados_test-osd-inprogress-rebalance.yaml:123]: Can we have the values of k, m = (8, 3) ? This and (8, 4) are the widely used configs for EC pool.

File: suites/pacific/rados/tier-3_rados_test-osd-inprogress-rebalance.yaml [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+              plugin: jerasure
+              rados_put: true
+          - create_pool:
+              create: true
+              pool_name: ercpool_2
+              pool_type: erasure
+              rados_put: true
+              pg_num: 32
+              pgp_num: 32
+              k: 3
```
Comment (pdhiran) [inline suites/pacific/rados/tier-3_rados_test-osd-inprogress-rebalance.yaml:134]: can we change k, m = (8, 4) here?

File: tests/rados/rados_test_util.py [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+    pools = config.get("create_pools")
+    for each_pool in pools:
+        cr_pool = each_pool["create_pool"]
+        if cr_pool.get("rados_put", False):
+            do_rados_put(mon=client_node, pool=cr_pool["pool_name"], nobj=100)
+        else:
+            method_should_succeed(rados_obj.bench_write, **cr_pool)
+
+
+def should_not_be_empty(variable, msg="Variable is empty"):
```
Comment (pdhiran) [inline tests/rados/rados_test_util.py:55]: This method can go to utils module.

**Review by pdhiran — CHANGES_REQUESTED** [review]
Minor changes in the suite file

File: suites/pacific/rados/tier-3_rados_test-osd-inprogress-rebalance.yaml (hunk 1 above)
Comment (tintumathew10) [reply suites/pacific/rados/tier-3_rados_test-osd-inprogress-rebalance.yaml]: modified and tested.

File: suites/pacific/rados/tier-3_rados_test-osd-inprogress-rebalance.yaml (hunk 2 above)
Comment (tintumathew10) [reply suites/pacific/rados/tier-3_rados_test-osd-inprogress-rebalance.yaml]: done

**Comment by pdhiran** [comment]
Do we have the pass logs after the changes?

**Pull request merged**